File: SSP.PureInsuranceRestAPIHandler/SSP.PureInsuranceRestAPIHandler/BaseClasses/BaseRiskRatingSectionType.cs
using SSP.PureInsuranceRestAPIHandler.Enums;

namespace SSP.PureInsuranceRestAPIHandler.BaseClasses
{
    public class BaseRiskRatingSectionType
    {
        public RowAction ActionType { get; set; }
        public string RatingSectionTypeCode { get; set; }

        public int RatingSectionTypeId { get; set; }

        public int SequenceNumber { get; set; }

        public string RateTypeCode { get; set; }

        public int RateTypeID { get; set; }

        public double AnnualRate { get; set; }

        public double SumInsured { get; set; }

        public bool SumInsuredSpecified { get; set; }

        public double AnnualPremium { get; set; }

        public bool AnnualPremiumSpecified { get; set; }

        public double ThisPremium { get; set; }

        public bool ThisPremiumSpecified { get; set; }

        public string CountryCode { get; set; }

        public int CountryID { get; set; }

        public string StateCode { get; set; }

        public int StateID { get; set; }

        public bool OriginalFlag { get; set; }
    }
}

File: Orion/Components/ImportExport/Interface/SiriusImport/RestContracts/BaseRiskRatingSectionType.vb
''' <summary>
''' Rating section data for a risk — maps to PureInsurance.REST.Common.Domain.Models.BaseRiskRatingSectionType
''' </summary>
Public Class BaseRiskRatingSectionType
    Public Property RatingSectionTypeCode As String
    Public Property RatingSectionTypeId As Integer
    Public Property SequenceNumber As Integer
    Public Property RateTypeCode As String
    Public Property RateTypeID As Integer
    Public Property AnnualRate As Double
    Public Property SumInsured As Double
    Public Property SumInsuredSpecified As Boolean
    Public Property AnnualPremium As Double
    Public Property AnnualPremiumSpecified As Boolean
    Public Property ThisPremium As Double
    Public Property ThisPremiumSpecified As Boolean
    Public Property CountryCode As String
    Public Property CountryID As Integer
    Public Property StateCode As String
    Public Property StateID As Integer
    Public Property OriginalFlag As Boolean
End Class
